.travis.yml

```diff
@@ -29,12 +29,9 @@ install:
     - conda config --set show_channel_urls true
     - conda config --set auto_update_conda false
 
-    # Now do the things we need to do to install it.
-    - conda install conda-execute --yes --quiet -c conda-forge
-
 script:
     - echo "Updating docs";
-    - source ./.ci_scripts/update_docs;
+    - bash ./ci_scripts/update_docs;
```

Comment:
Should this be `bash ./.ci_scripts/update_docs;`?

Reply:
yes 😞